README.md

```diff
@@ -1,5 +1,11 @@
 # InSpec: Inspect Your Infrastructure
 
+* **Project State: Active**
+* **Issues Response SLA: 3 business days**
```

Comment:
What does a "response" entail?

Reply:
https://github.com/chef/chef-oss-practices/blob/master/repo-management/repo-states.md#what-is-the-response-time-maximum

`
The Response Time Maximum (RTM) applies to responses from a project member to issues or pull requests. It does not apply to the resolution of an issue or the merging of a pull request. Resolution of an issue or merging of a pull request is highly variable depending on what is involved in them.
`